trade-finance-manager-ui/component-tests/case/parties/_macros/parties-bond-beneficiary-edit.component-test.js

```diff
@@ -8,7 +9,7 @@ describe(page, () => {
 
   describe('bond issuer', () => {
     const params = {
-      bondType: 'bond issuer',
```

Comment:
Assume this was nonsense data & thats why its been replaced

Reply:
Correct, there is no bond type called "bond issuer", should all be coming from `BOND_FACILITY_TYPE` now.